packages/react/src/components/ThemeProvider/index.tsx

```diff
@@ -1,23 +1,42 @@
 import * as React from 'react';
+import { DirectionProvider } from '@radix-ui/react-direction';
 
 import { createTheme, Theme } from '@aws-amplify/ui';
 
 import { AmplifyContext } from './AmplifyContext';
 
 export type ColorMode = 'system' | 'light' | 'dark';
+export type Direction = 'ltr' | 'rtl';
 
 interface ThemeProviderProps {
   children: React.ReactNode;
+  /**
+   * Changes the color mode applied to the theme
```

Comment:
Should we add `@description` tags to these props?
```suggestion
   * @description Changes the color mode applied to the theme
```